### ISSUE TITLE
Provide ability to specify search order in the Web client URL

### ISSUE THREAD

**Issue by shmerl** [bluesky-social/social-app#4489]
Currently, the Web client URL for search looks like this:

```
https://bsky.app/search?q=<search_term>
```

Only after you get the results you sort them by top / latest / people.

Please add the ability to specify sort order in that top URL itself, something like
```
https://bsky.app/search?q=<search_term>&sort=latest
```

which would be passed to the underlying feed query which already supports it.

**Comment by mary-ext** [comment]
Duplicate of https://github.com/bluesky-social/social-app/issues/4073

**Comment by shmerl** [comment]
Ah, I'll close this then.
